resources/assets/pages/ShowCampaign.js

```diff
@@ -57,6 +58,23 @@ const ShowCampaign = () => {
     );
   }
 
+  if (status === 'signups') {
+    return (
+      <Shell title={title} subtitle={data.campaign.internalTitle}>
+        <div className="container__row">
+          <div className="container__block text-right pt-heading">
+            <Link className="button -tertiary" to={`/campaigns/${id}`}>
```

Comment:
Is there a reason we're not using more of these `Link` components vs `<a>` tags? I know we need it for linking to the create/edit forms, but I'd imagine using a `Link` would be preferable for when we're linking to React routes?

Reply:
Yeah! If there's anywhere we're linking to a client-side route with a plain 'ol `a`, that's defs a mistake.